src/_includes/macros/active-filters.html

```diff
@@ -0,0 +1,143 @@
+
+{% macro render(posts, filters, options) %}
+
+{% set active_filters = [] %}
+{% set active_date_filters = [] %}
+{% set use_list = options.use_list or false %}
+
+{# This is a hack. `_ignore` is a variable we will never use but we can use
+   its var expression as a means to write the expression needed to update
+   `active_date_filters`. #}
+{%- for filter in filters %}
+    {%- if filter == 'range_date' -%}
+        {% set _ignore = active_date_filters.extend(selected_filters_for_field('range_date_gte')) %}
+        {% set _ignore = active_date_filters.extend(selected_filters_for_field('range_date_lte')) %}
+    {%- elif filter == 'activity_category' -%}
+        {% set _ignore = active_filters.extend(selected_filters_for_field('type')) %}
+        {% set _ignore = active_filters.extend(selected_filters_for_field('category')) %}
+    {%- else -%}
+        {% set _ignore = active_filters.extend(selected_filters_for_field(filter)) %}
+    {%- endif -%}
+{% endfor -%}
+
+{%- set active_filters_total = active_filters|length + active_date_filters|length -%}
+
+{%- if active_filters_total > 0 and use_list == false -%}
+  {{ _active_filters_notification() }}
+{% elif use_list == true %}
+  {{ _active_filters_list(active_filters_total, active_filters, active_date_filters, options) }}
+{% endif %}
+
+{% endmacro %}
+
+
+{# A helper macro to display the active_filters as a list.
+  ========================================================================== #}
+{% macro _active_filters_list(active_filters_total, active_filters, active_date_filters, options) %}
+{%- if active_filters_total > 0 -%}
+    <ul class="filtered-by filtered-by__align-with-btn list__horizontal">
+        {%- if options.show_filters_label -%}
+        <li class="list_item filtered-by_header">
+        {%- if posts|list|length > 0 -%}
+            {{ posts.total }} filtered result{{ 's' if posts.total > 1 }} for
+        {%- else -%}
+            No results for
+        {%- endif -%}
+        </li>
+        {%- endif %}
+    {%- for filter in active_filters %}
+        <li class="list_item filtered-by_filter">
+            {{ 'Blog' if filter == 'post' else filter }}
+        {%- if not loop.last -%}
+            ,
+        {%- elif active_date_filters|length > 0 -%}
+            ,
+        {%- endif %}
+        </li>
+    {% endfor %}
+    {%- if active_date_filters|length > 0 -%}
+        <li class="list_item filtered-by_filter">
+        {%- for filter in active_date_filters %}
+            {{ filter|date("%B %Y") }}
+            {{ '&ndash;'|safe if not loop.last }}
+        {%- endfor -%}
+        </li>
+    {%- endif %}
+    </ul>
+{%- else -%}
+    {% if options.show_unfiltered_count and options.show_filters_label %}
+    <ul class="filtered-by filtered-by__align-with-btn list__horizontal">
+        <li class="list_item">
+            <span class="filtered-by_header">
+                {{ posts.total }} results
```

Comment:
Will `posts.total` always be greater than 1? Otherwise include logic to have `result` and `results`.